readthedocs_build/config/test_config.py

```diff
@@ -15,7 +15,6 @@
 from .config import PYTHON_INVALID
 from .validation import INVALID_BOOL
 from .validation import INVALID_CHOICE
-from .validation import INVALID_DIRECTORY
```

Comment:
Let's keep this line for now, in case we have some side effects from removing a line like this.

Reply:
Done!